docs/reference/commandline/images.md

```diff
@@ -343,3 +344,39 @@ b6fa739cedf5        committ                   latest
 746b819f315e        postgres                  latest
 {% endraw %}
 ```
+
+## Customizing display orders
+
+To customize list orders, you can use `--sort-by` option, this option takes format
+of `field[:order]`. `field` should be a case insensitive string among "repository",
+"tag", "digest", "createdat", "id" and "size". `order` should be either `asc`(ascendant)
+or `desc`(descendant).
+The meaning of each `field` is described in [`Formatting`](#formatting) section.
+
+If you specify a wrong `field` or `order` string, it will output the right options
```

Comment:
s/a wrong/an invalid

s/it will output...options/a hint will be printed, such as the following: